src/templates/getPageResolver.ts

```diff
@@ -0,0 +1,32 @@
+import { posix } from 'path'
+
+import { outdent } from 'outdent'
+import slash from 'slash'
+import glob from 'tiny-glob'
+
+import { HANDLER_FUNCTION_NAME } from '../constants'
+
+// Generate a file full of require.resolve() calls for all the pages in the
+// build. This is used by the nft bundler to find all the pages.
+
+export const getPageResolver = async ({ netlifyConfig, target }) => {
```

Comment:
[sand] Same, can be typed here as a follow up.

Good indicator that we should enable the eslint rule for typing functions, I'll add that to the board

Reply:
Yeah, good point. I'd say add the one to require it at module boundaries at least